dev.Dockerfile

```diff
@@ -1,15 +1,15 @@
-# This is a Dockerfile for Developer Container
+# This is a Dockerfile for the Development Container
 
 # Use the Python base image
 ARG VARIANT="3.11-bullseye"
-FROM mcr.microsoft.com/vscode/devcontainers/python:0-${VARIANT} AS langchain-dev-base
+FROM mcr.microsoft.com/devcontainers/python:0-${VARIANT} AS langchain-dev-base
```

Comment:
FYI, we will soon release a new major version of python in order to support `debian:bookworm`
https://github.com/devcontainers/images/issues/622

I'd recommend switching once it's live!

Reply:
Thanks for letting us know, @samruddhikhandale !